src/components/Expenses/NewExpense/NewExpense.tsx

```diff
@@ -1,10 +1,16 @@
 import React, { useState } from "react";
+import { expenseData } from "../../../App";
 import ExpenseForm from "./ExpenseForm";
 import "./NewExpense.css";
 
-const NewExpense = ({ onNewExpenseRender }) => {
-  const [displayExpenseForm, updateDisplayExpenseForm] = useState(false);
-  const onNewExpense = (expenseObj) => {
+const NewExpense = ({
+  onNewExpenseRender,
+}: {
+  onNewExpenseRender: (arg: expenseData) => void;
+}) => {
+  const [displayExpenseForm, updateDisplayExpenseForm] =
+    useState<boolean>(false);
+  const onNewExpense = (expenseObj: expenseData) => {
     const newExpense = {
       ...expenseObj,
       id: Math.random().toString(),
```

Comment:
For `id` variables, it's always best to use `uuid` or `cuid`. They make guessing the id of the element nearly impossible and thus add a higher degree of security.